x-pack/plugins/profiling/public/components/topn_functions.tsx

```diff
@@ -57,6 +58,18 @@ function CPUStat({ cpu, diffCPU }: { cpu: number; diffCPU: number | undefined })
   );
 }
 
+function TotalDiff({ samples1, samples2 }: { samples1: number; samples2: number }) {
+  if (samples1 === samples2 || samples1 === 0) {
+    return <></>;
+  }
+
+  const diff = Math.abs(1 - samples2 / samples1) * 100;
+  const text = (samples1 < samples2 ? '+' : '-') + `${diff.toFixed(2)}%`;
+  const color = samples1 < samples2 ? 'danger' : 'success';
+
+  return <EuiTextColor color={color}> {text}</EuiTextColor>;
```

Comment:
Could you add parentheses around `{text}`? It helps visually distinguish the value and its percentage, but it also matches the look in the rightmost CPU columns.

Reply:
Fixed